.github/pull_request_template.md

```diff
@@ -1,39 +1,33 @@
 ## Description
+<!--- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
```

Comment:
Nitpicking
```suggestion
<!-- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
```

Reply:
Well, ignore me. I was not sophisticated enough to know that triple dash could mean something special.
https://stackoverflow.com/questions/4823468/comments-in-markdown#comment17954715_4829998

Reply:
Nah, maybe I'm being too smart here. So let's avoid surprises for everyone, especially given [GitHub mentions two dashes](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#hiding-content-with-comments). I'll fix this.